nox/_parametrize.py

```diff
@@ -32,70 +82,83 @@ def parametrize_decorator(arg_names, arg_values_list):
             argument names were specified, this must be a list of N-tuples,
             where each tuple-element specifies a value for its respective
             argument name, for example ``[(1, 'a'), (2, 'b')]``.
+        ids ([Sequence[str]): Optional sequence of test ids to use for the
```

Comment:
1. `[Sequence[str]` -> `Optional[[Sequence[str]]` or `Sequence[str]` (i.e. you have an unmatched `[`)
1. s/ids/IDs (initialisms grumble grumble)

Reply:
1. Fixed unbalanced brackets, but parameters with default args are always implied Optional.
2. Done. :)